src/shipit_code_coverage/shipit_code_coverage/report_generators.py

```diff
@@ -33,25 +31,19 @@ def zero_coverage(artifacts, out_dir='code-coverage-reports'):
                     zero_coverage_functions[name].append(f['name'])
                 else:
                     zero_coverage_functions[name] = [f['name']]
-            else:
-                all_functions_uncovered = False
-
-        if all_lines_uncovered or (len(sf['functions']) > 1 and all_functions_uncovered):
-            zero_coverage_files.append(name)
 
-    with open(os.path.join(out_dir, 'zero_coverage_files.json'), 'w') as f:
-        json.dump(zero_coverage_files, f)
-
-    mkdir(os.path.join(out_dir, 'zero_coverage_functions'))
+    mkdir(os.path.join(out_dir, 'zero_coverage_reports'))
```

Comment:
Only the file `zero_coverage_functions.json` should be renamed to `zero_coverage_reports.json`, the directory shouldn't change.